### PR TITLE
Use `--skip` to set Bitbucket offset.

### LINKED ISSUE #76: start option is poorly named
The name of the start option suggests that it's an issue number from which to start, when it's actually an offset to skip past. Although computer scientists hold it as a badge of honor to start numbering with zero, but when the issues are numbered starting with 1, the interface for the issue migration should do the same. Start should indicate the first issue to be processed or it should be renamed to something like "skip" or "offset".

### PR DESCRIPTION
**Pull request by jaraco**
Fixes #76.

### CODE REVIEW

**Comment by jeffwidman** [comment]
(thumbsup)

**Comment by jeffwidman** [comment]
Actually, can you update README as well?

File: migrate.py [hunk 1]
```diff
@@ -79,14 +79,10 @@ def read_arguments():
     )
 
     parser.add_argument(
-        "-f", "--start", type=int, dest="start", default=0,
+        "-f", "--skip", type=int, default=0,
```
Comment (jeffwidman) [inline migrate.py]: Do you not need `dest="skip"`?

Comment (jaraco) [inline migrate.py]: Nah. Dest defaults to the long option.

**Comment by jaraco** [comment]
I'll update the REadme

**Comment by jeffwidman** [comment]
Thanks!

**Pull request merged**